index.html

```diff
@@ -732,11 +732,35 @@ <h2>Data Model</h2>
 many, but not all, of the concepts described in those other specifications.
 </p>
 <p>
-The unified data model for this work can be represented as a list of entries
-that represent parties.
+The unified data model for this work can be represented as a list of service providers
+that represent entities or orgnaizations that provides services such as credential issuance or validation. The Data Model also contains the details of the list operator informations.
 </p>
+
+<h1>Overview of List Operator</h1>
+<p>The List Operator provides a framework for managing a trusted list, which includes essential information about the entity responsible for the list's maintenance and its associated attributes.</p>
+
+<h2>Attributes of List Operator</h2>
+<ul>
+    <li><strong>ListOperatorInformation:</strong> Provides a set of information about the owner of the trusted list.</li>
```

Comment:
Should this not be a list of `<dfn>` entries? And probably alpha-ordered?

Reply:
The list operator information is a large set of information about the LO. Some elements of this will be registered names, others will not be. So which element do you want to be <dfn>?